ivr/src/main/resources/webapp/partials/settings.html

```diff
@@ -25,10 +25,11 @@ <h4 class="frame-title">{{msg('ivr.web.settings.title')}}</h4>
     <div class="clearfix">
         <form name="form" class="form-horizontal" novalidate>
             <div class="panel-group">
-                <div class="panel panel-default ivr-config" ng-repeat="conf in configs">
+                <div class="panel panel-default ivr-config" ng-repeat="conf in configs.configList">
                     <div class="panel-heading" ng-click="accordions[$index] = !accordions[$index]">
                         <i ng-class="{'fa-caret-down': accordions[$index], 'fa-caret-right': !accordions[$index]}" class="fa fa-fw fa-lg pointer"></i>
                         {{conf.name}}
+                            <i ng-show="configs.defaultConfig == conf.name" data-toggle="tooltip" data-placement="top" data-trigger="hover" bs-popover="{{msg('ivr.settings.defaultConfig.tooltip')}}" class="fa fa-star"></i>
```

Comment:
Maybe use index, when i was editing non default configuration i had two stars when i was providing name(name of the second config has the same prefix), example: 'conf' and 'conf2'. Also this tooltip is not fully visible when the name is short.